### PR TITLE
games-engines/fheroes2: new package

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

**Comment by SoapGentoo** [comment]
is there a package that requires this?

**Comment by DarthGandalf** [comment]
@SoapGentoo no, I discussed it with @chewi on #gentoo-games prior to writing this, and the way how it's split between packages

File: games-engines/fheroes2/fheroes2-1.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop
```
Comment (chewi) [inline games-engines/fheroes2/fheroes2-1.0.4.ebuild:6]: Does this actually use the desktop eclass? It should use the xdg eclass though to update the icon cache.

File: games-engines/fheroes2/fheroes2-1.0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	media-libs/libsdl2[video]
+	media-libs/sdl2-image
+	media-libs/sdl2-mixer
+	sys-libs/zlib
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	app-arch/libarchive
+	virtual/libintl
+"
```
Comment (chewi) [inline games-engines/fheroes2/fheroes2-1.0.4.ebuild:28]: I think you need to add `dev-lang/python` here for `extract_homm2_resources.sh` ? Technically, we should only support specific versions, but that might be overkill when it's only for a few short lines.

**Review by chewi — CHANGES_REQUESTED** [review]
I've tried the demo, seems to work fine. I've also bought the game from GOG, as it has a heavy discount right now, but no time to try it tonight.

File: games-engines/fheroes2/fheroes2-1.0.4.ebuild (hunk 2 above)
Comment (DarthGandalf) [reply games-engines/fheroes2/fheroes2-1.0.4.ebuild]: Done

File: games-engines/fheroes2/fheroes2-1.0.4.ebuild (hunk 1 above)
Comment (DarthGandalf) [reply games-engines/fheroes2/fheroes2-1.0.4.ebuild]: Done

**Pull request closed without merging**